views/content-cross-linked.phtml

```diff
@@ -1,3 +1,16 @@
 <?= $data->content ?>
 <hr>
-<p><em>Also published on <a href="<?= $data->medium_post->url ?>">Medium</a>.</em></p>
+<p>
+	<em>
+		<?php
+		printf(
```

Comment:
would prefer to keep this on two lines:

```php
$link = sprintf('<a href="%s">%s</a>', esc_url($data->medium_post->url), __("Medium", "medium"));
printf(_x("Also published on %s.", "Medium post link", "medium"), $link);
```

I'll add this to the readme, but in general, style-wise, I'm going to request that patches follow the style of the existing plugin code:

* Double quotes for all strings by default. Single quotes for strings that include `"`, but do not include `'`.
* No braces for single line if statements.
* underscored_variables.
* No spaces inside parentheses.